### PR TITLE
Update nx-plugin interface descriptions to match docs

### PR DESCRIPTION
**Pull request by emlyn-rice**
Fix location bug with executor

### CODE REVIEW

File: libs/nx-plugin/src/executors/code-gen/schema.json [hunk 1]
```diff
@@ -33,15 +33,15 @@
...
             "type": "string",
-            "description": "Sub-folder to add the generated code to.",
+            "description": "The Sub-folder of the project to add the generated code to.",
             "default": "./"
         },
 
         "dryRun": {
             "type": "boolean",
-            "description": "Only show what would be generated.",
+            "description": "Only show what would be generated don't modify files.",
```
Comment (MatthewJNixon) [inline libs/nx-plugin/src/executors/code-gen/schema.json]: I think this needs some punctuation here?

Comment (emlyn-rice) [reply libs/nx-plugin/src/executors/code-gen/schema.json]: Fixed

File: libs/nx-plugin/generators.json [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
             "factory": "./src/generators/code-gen/generator",
             "schema": "./src/generators/code-gen/schema.json",
-            "description": "Generates FeatureBoard types for your chosen language"
+            "description": "Generate code based on the features set in FeatureBoard"
         },
         "add-code-gen-target": {
             "factory": "./src/generators/add-code-gen-target/generator",
             "schema": "./src/generators/add-code-gen-target/schema.json",
-            "description": "Adds code gen to a projects targets"
+            "description": "Adds the code gen executor to a projects targets"
```
Comment (karimoran) [inline libs/nx-plugin/generators.json]:
```suggestion
            "description": "Adds the code-gen executor to the project target."
```

File: libs/nx-plugin/src/executors/code-gen/schema.json [hunk 3]
```diff
@@ -33,15 +33,15 @@
 
         "subFolder": {
             "type": "string",
-            "description": "Sub-folder to add the generated code to.",
+            "description": "The Sub-folder of the project to add the generated code to.",
```
Comment (karimoran) [inline libs/nx-plugin/src/executors/code-gen/schema.json]:
```suggestion
            "description": "The target sub-folder of the project.",
```

File: libs/nx-plugin/src/generators/code-gen/schema.json [hunk 4]
```diff
@@ -46,7 +47,7 @@
 
         "subFolder": {
             "type": "string",
-            "description": "Sub-folder to add the generated code to.",
+            "description": "The Sub-folder of the project to add the generated code to.",
```
Comment (karimoran) [inline libs/nx-plugin/src/generators/code-gen/schema.json]:
```suggestion
            "description": "The target sub-folder of the project.",
```

File: libs/nx-plugin/executors.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
         "code-gen": {
             "implementation": "./src/executors/code-gen/executor",
             "schema": "./src/executors/code-gen/schema.json",
-            "description": "Generates FeatureBoard types for your chosen language"
+            "description": "Generate code based on the features set in FeatureBoard"
```
Comment (karimoran) [inline libs/nx-plugin/executors.json]:
```suggestion
            "description": "Generate code based FeatureBoard configuration."
```

File: libs/nx-plugin/generators.json [hunk 6]
```diff
@@ -3,12 +3,12 @@
         "code-gen": {
             "factory": "./src/generators/code-gen/generator",
             "schema": "./src/generators/code-gen/schema.json",
-            "description": "Generates FeatureBoard types for your chosen language"
+            "description": "Generate code based on the features set in FeatureBoard"
```
Comment (karimoran) [inline libs/nx-plugin/generators.json]:
```suggestion
            "description": "Generate code based on FeatureBoard configuration."
```

File: libs/nx-plugin/src/generators/add-code-gen-target/schema.json [hunk 7]
```diff
@@ -48,7 +49,7 @@
 
         "subFolder": {
             "type": "string",
-            "description": "Sub-folder to add the generated code to.",
+            "description": "The sub-folder of the project to add the generated code to.",
```
Comment (karimoran) [inline libs/nx-plugin/src/generators/add-code-gen-target/schema.json]:
```suggestion
            "description": "The target sub-folder of the project .",
```

File: libs/nx-plugin/src/generators/code-gen/schema.json [hunk 8]
```diff
@@ -26,17 +26,18 @@
...
                 "$source": "projectName"
             },
-            "x-prompt": "What is the name of the project you want to add the component to?"
+            "x-prompt": "What is the name of the project you want to add the generated code to?"
         },
 
         "featureBoardProductName": {
             "type": "string",
-            "description": "The name of the FeatureBoard product you want to pull the types from."
+            "description": "The name of the FeatureBoard product you want to pull the types from.",
```
Comment (karimoran) [inline libs/nx-plugin/src/generators/code-gen/schema.json]:
```suggestion
            "description": "The FeatureBoard product to pull the types from.",
```

**Review by karimoran — COMMENTED** [review]
@emlyn-arkahna I've provided some alternatives that are a little more succinct, however I may be misunderstanding some of the instructions here so let me know if they now don't make sense.

File: libs/nx-plugin/executors.json [hunk 9]
```diff
@@ -3,7 +3,7 @@
         "code-gen": {
             "implementation": "./src/executors/code-gen/executor",
             "schema": "./src/executors/code-gen/schema.json",
-            "description": "Generates FeatureBoard types for your chosen language"
+            "description": "Generate code based FeatureBoard configuration."
```
Comment (karimoran) [inline libs/nx-plugin/executors.json]:
```suggestion
            "description": "Generates code based on FeatureBoard configuration."
```

**Pull request merged**